collectors/python.d.plugin/isc_dhcpd/README.md

```diff
@@ -12,6 +12,7 @@ Monitors the leases database to show all active leases for given pools.
 
 -   dhcpd leases file MUST BE readable by Netdata
 -   pools MUST BE in CIDR format
+-   'python-ipaddress' package is needed
```

Comment:
this is true only for py2, right?

Reply:
Yes. I have been testing with py2 and py3 and it is only needed in py2.